sgt-back-end/server.js

```diff
@@ -0,0 +1,129 @@
+import express from './node_modules/express/index.js';
+
+import pg from 'pg';
+
+const db = new pg.Pool({
+  connectionString: 'postgres://dev:dev@localhost/studentGradeTable',
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+const app = express();
+
+app.use(express.json());
+
+app.get('/api/grades', async (req, res) => {
+  try {
+    const sql = `
+    select *
+    from "grades";`
+    const result = await db.query(sql)
+    const grades = result.rows;
+      res.status(200).json(grades);
+    } catch (err) {
+    console.error('/api/notes error', err);
+    res.status(500).json({ error: 'an unexpected error occurred' });
+  }});
+
+app.post('/api/grades', async (req, res) => {
+  try {
+    const name = req.body.name;
+    const course = req.body.course;
+    const score = +req.body.score;
+    const params = [name, course, score];
+    const sql = `
```

Comment:
You don't use this until way down at line 49. The code would read better if you defined this down there where you use it. Same with `params`.